content/docs/04.5-devx.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Palette Dev Engine"
+metaTitle: "Palette Dev Engine "
+metaDescription: "Explore Palette Dev Engine"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Palette provides a developer persona of the Palette operations in addition to the DevOps persona of creating, deploying, and managing applications in Palette. This persona is called Palette Dev Engine (PDE); the experience developers have while building, maintaining, testing, deploying, and monitoring applications through Palette. A developer can easily switch between these two personas - `App Mode and Cluster Mode`- through the Palette UI. We offer the PDE from two wide perspectives:
+
```

Comment:
Palette now makes it easy for app developers to model and deploy cloud-native applications at scale. <-- this is simpler. PDE is the marketing term and not a persona name.

What are the two wide perspectives?

Reply:
added 
* App Mode: The developer deploys applications rapidly.

* Cluster Mode: To deploy and manage Palette clusters and explore the devops persona of Palette.